Carmu/Sources/Extensions/String+Extension.swift

```diff
@@ -21,4 +21,9 @@ extension String {
 
         return modifiedSubtitle.trimmingCharacters(in: .whitespaces)
     }
+
+    // 문자열의 앞글자만 대문자로 바꿔주는 메서드 (프로필 이미지 초기화에 필요)
+    func capitalizedFirstCharacter() -> String {
```

Comment:
<img width="741" alt="image" src="https://github.com/DeveloperAcademy-POSTECH/MacC-Team-DGC/assets/22979718/f21648f4-0e15-40ff-896c-3873c70b5858">

요렇다고 합니다 😅

```suggestion
    func capitalized() -> String {
```

Reply:
오호...